SmartDeviceLink/SDLProtocol.m

```diff
@@ -310,9 +335,11 @@ - (BOOL)sendRPC:(SDLRPCMessage *)message encrypted:(BOOL)encryption error:(NSErr
             // If we're trying to encrypt, try to have the security manager encrypt it. Return if it fails.
             // TODO: (Joel F.)[2016-02-09] We should assert if the service isn't setup for encryption. See [#350](https://github.com/smartdevicelink/sdl_ios/issues/350)
             messagePayload = encryption ? [self.securityManager encryptData:rpcPayload.data withError:error] : rpcPayload.data;
```

Comment:
When I attempt to send an encrypted `AddCommand` it crashes here with a `EXC_BAD_ACCESS` error. Any idea what might be happening? Not sure if it is an issue with the library or the security manager.